### PR TITLE
Unused CSS File Removal PR 2

### PR DESCRIPTION
**Pull request by Isha988**
## Related Issue
Follow up for  #568 

## Description of Changes

Started Removing unused file CSS files of public folder

### CODE REVIEW

**Comment by Isha988** [comment]
Pushing small commit just to get a range of deployments.

**Comment by Isha988** [comment]
@amode @muhammadjufry do we need `public/assets/css/animate.min.css` and `public/assets/css/animate.min.css` files any more?

**Comment by Isha988** [comment]
Removing bootstrap CSS file is causing problem in contact page, and maybe some other pages too using bootstrap.
So, do I remove it and fix the issue so arise or focus on something else more important.

And please for once check that removed files are not causing any problem.

**Comment by anmode** [comment]
> @amode @muhammadjufry do we need `public/assets/css/animate.min.css` and `public/assets/css/animate.min.css` files any more?

Nope.

**Comment by anmode** [comment]
> Removing bootstrap CSS file is causing problem in contact page, and maybe some other pages too using bootstrap. So, do I remove it and fix the issue so arise or focus on something else more important.
> 
> And please for once check that removed files are not causing any problem.

abhi toh theek lg rha h... contact page.. But no worries. Will replace and fix if problem aai. Maine sb dekh lia abhi toh theek h UI.

File: pages/contact.page.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ import * as yup from "yup";
 import { useFormik } from "formik";
 import { error } from "jquery";
```
Comment (anmode) [inline pages/contact.page.js:10]: ok so you fixed??

**Review by anmode — APPROVED** [review]

**Comment by Isha988** [comment]
> > Removing bootstrap CSS file is causing problem in contact page, and maybe some other pages too using bootstrap. So, do I remove it and fix the issue so arise or focus on something else more important.
> > And please for once check that removed files are not causing any problem.
> 
> I'll make an issue to use Tailwind or not to use Bootstrap. Maine sb dekh lia abhi toh theek h UI. Wow kaafi files reduce hogyi...

Actually, I haven't removed bootstrap till now.
Really destruction will be after removing it.

**Comment by anmode** [comment]
> > > Removing bootstrap CSS file is causing problem in contact page, and maybe some other pages too using bootstrap. So, do I remove it and fix the issue so arise or focus on something else more important.
> > > And please for once check that removed files are not causing any problem.
> > 
> > 
> > I'll make an issue to use Tailwind or not to use Bootstrap. Maine sb dekh lia abhi toh theek h UI. Wow kaafi files reduce hogyi...
> 
> Actually, I haven't removed bootstrap till now. Really destruction will be after removing it.

haaa abhi rhndo.. Dheere Dheere pehle codebase se remove krenge... phir hta denge files

**Pull request merged**